calicoctl/resourcemgr/resourcemgr.go

```diff
@@ -399,6 +400,7 @@ func unmarshalResource(tm unstructured.Unstructured, b []byte) ([]runtime.Object
 		return nil, err
 	}
 
+	defaults.SetDefaults(unpacked)
```

Comment:
I'm a little bit hesitant to introduce a new library here - to be honest, I feel like this must already be working for the equivalent ipipMode field which has the same semantics. I haven't had time to dig into it yet, but feel like this should behave the same as the `ipipMode` field does.

Reply:
@caseydavenport I added a test for missing IpipMode field and I am seeing this error: `error with field IPpool.IPIPMode = '' (IPIPMode other than 'Never' is not supported on an IPv6 IP pool)`